README.md

```diff
@@ -103,14 +115,14 @@ class MyDomainClass {
 
     static hasMany = [categories: Category, items: Item]
 
-    static search = {
+    static luceneIndexing = {
```

Comment:
I think the former name was better, since Hibernate Search could rely on other engine than lucene (such as ElasticSearch) and we don't want all users to rename their closures :) Could you switch back to "search" please?

Reply:
Oh, I see. I think the point was actually to have the same name to make it simple to use and understand, but if you really think it could collide in some way, I would then vote for hibernateSearchIndexing.

Honestly, I gave up on the Groovy tooling (mostly because I use Eclipse / cannot afford Intellij :)), I just try to use @GrailsCompileStatic as much as I can, but since we are using a dynamically added methods here, I don't think we can do anything. I am not aware of such dynamic descriptors (such as grails plugins describing which methods they could add) but I doubt it exists, and even if, not sure that tooling would use it.

Anyway, thanks for renaming, I will merge after it, and try on other project

Reply:
The collision was most noticeable when running application in live-reload, as then you'd get errors saying "cannot apply list()" to xxxx. That error is actually solved by the updates to the plugin class and the use of onChange, so the actual collision is now gone.

So i'll name it back to `search` and we've got the section in the readme `IDE Integration` to help tooling